packages/atlas/src/components/_crt/RevenueShareStateWidget/RevenueShareStateWidget.tsx

```diff
@@ -111,32 +116,42 @@ export const RevenueShareStateWidget = ({
             ? 'REVENUE SHARE STATE'
             : status === 'past'
             ? 'REVENUE SHARE ENDED ON'
+            : status === 'upcoming'
+            ? 'REVENUE SHARE STARTS IN'
             : 'REVENUE SHARE ENDS IN'
         }
         customNode={
-          status !== 'inactive' && endsAt ? (
-            <FlexBox justifyContent="space-between" alignItems="center" width="100%">
-              {status === 'past' ? (
-                <Text variant="h500" as="h5" margin={{ bottom: 4 }}>
+          <FlexBox justifyContent="space-between" alignItems="center" width="100%">
+            {status === 'inactive' ? (
+              <Text variant="h500" as="h5" margin={{ bottom: 4 }}>
+                No active share
+              </Text>
+            ) : status === 'past' ? (
+              <Text variant="h500" as="h5" margin={{ bottom: 4 }}>
+                {endingDate ? formatDateTime(endingDate).replace(',', ' at') : 'N/A'}
+              </Text>
+            ) : status === 'upcoming' ? (
+              <FlexBox flow="column">
+                <Text variant="h500" as="h5">
+                  {startingDate ? formatDurationShort(Math.round((startingDate.getTime() - Date.now()) / 1000)) : 'N/A'}
+                </Text>
+                <Text variant="t100" as="p" color="colorText">
+                  {startingDate ? formatDateTime(startingDate).replace(',', ' at') : 'N/A'}
```

Comment:
Replace with `formatDateTimeAt` additionally introduce a component that will handle count down
- performance improvement - react not rerender whole component 
- encapsulate the logic when the timer turns red